calibration/CTDMOG/CGINS-CTDMOG-13425__20161020.csv

```diff
@@ -22,4 +22,4 @@ serial,name,value,notes
 37-13425,CC_ptempa0,1.355180e+002,
 37-13425,CC_ptempa1,-6.727311e-002,
 37-13425,CC_ptempa2,5.276720e-007,
-37-13425,CC_wbotc,1.8588e-007,
+37-13425,CC_wbotc,1.858827e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT